server/sonar-server/src/test/java/org/sonar/server/tester/UserSessionRule.java

```diff
@@ -102,22 +102,22 @@ public static UserSessionRule forServerTester(ServerTester serverTester) {
   /**
    * Log in with the default login {@link #DEFAULT_LOGIN}
    */
-  public UserSessionRule login() {
-    return login(DEFAULT_LOGIN);
+  public UserSessionRule log_in() {
```

Comment:
Introducing an underscore in a Java method does not improve clarity. Please rename to `logIn()`.